account_statement_import/migrations/14.0.2.0.0/pre-migration.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2022 ForgeFlow S.L.  <https://www.forgeflow.com>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    if openupgrade.column_exists(
+        env.cr, "account_bank_statement_line", "partner_bank_id"
+    ):
+        # between 14.0.1.0.0 and 14.0.2.0.0, lines had a partner_bank_id field
+        openupgrade.logged_query(
+            env.cr,
+            """
+            UPDATE account_move am
+            SET partner_bank_id = COALESCE(absl.partner_bank_id, am.partner_bank_id)
+            FROM account_bank_statement_line absl
+            WHERE am.statement_line_id = absl.id
```

Comment:
For not updating more rows than needed:
```suggestion
            UPDATE account_move am
            SET partner_bank_id = absl.partner_bank_id
            FROM account_bank_statement_line absl
            WHERE am.statement_line_id = absl.id
            AND am.partner_bank_id IS NULL AND absl.partner_bank_id IS NOT NULL
```